### PR TITLE
fix a desription error in sysctl file.

### PR DESCRIPTION
**Pull request by WanLinghao**
This patch fixes a error description on
docs/concepts/cluster-administration/sysctl-cluster.md

### CODE REVIEW

**Comment by WanLinghao** [comment]
@sttts @php-coder  PTAL

File: docs/concepts/cluster-administration/sysctl-cluster.md [hunk 1]
```diff
@@ -132,17 +132,19 @@ to schedule those pods onto the right nodes.
 
 The use of sysctl in pods can be controlled via annotations on the PodSecurityPolicy.
 
-Here is an example, it authorizes binding user creating pod with corresponding
-_safe_ and _unsafe_ sysctls.
+Sysctl annotations represents the key of a whitelist of allowed safe and unsafe sysctls
+in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
+(which end in * ). The string * matches all sysctls.
```
Comment (php-coder) [inline docs/concepts/cluster-administration/sysctl-cluster.md]: s/*/`*`/

Comment (php-coder) [reply docs/concepts/cluster-administration/sysctl-cluster.md]: In other words, please, surround asterisk by apostrophes.

File: docs/concepts/cluster-administration/sysctl-cluster.md [hunk 2]
```diff
@@ -132,17 +132,19 @@ to schedule those pods onto the right nodes.
 
 The use of sysctl in pods can be controlled via annotations on the PodSecurityPolicy.
```
Comment (php-coder) [inline docs/concepts/cluster-administration/sysctl-cluster.md]: s/annotations/annotation/

File: docs/concepts/cluster-administration/sysctl-cluster.md (hunk 1 above)
Comment (bradtopol) [reply docs/concepts/cluster-administration/sysctl-cluster.md]: Per the style guidelines, isn't the use of apostrophes around the asterisk not recommended? 
See https://deploy-preview-7014--kubernetes-io-master-staging.netlify.com/docs/home/contribute/style-guide/#use-normal-style-for-string-and-integer-field-values

Comment (php-coder) [reply docs/concepts/cluster-administration/sysctl-cluster.md]: Maybe you're right but it's not just a string. Also if you look at https://deploy-preview-7666--kubernetes-io-master-staging.netlify.com/docs/concepts/cluster-administration/sysctl-cluster/#podsecuritypolicy-annotations you see that now asterisk doesn't emphasized and looks not so good. In the PSP doc, asterisk is highlighted everywhere.

**Comment by zacharysarah** [comment]
@php-coder @bradtopol 👋 Because the text is talking about a code output, in this case I think it's OK to enclose the asterisks in back ticks (for inline code format).

So:
```
    (which end in `*` ). The string `*` matches all sysctls.
```

File: docs/concepts/cluster-administration/sysctl-cluster.md [hunk 3]
```diff
@@ -130,19 +130,21 @@ to schedule those pods onto the right nodes.
 
 ## PodSecurityPolicy Annotations
 
-The use of sysctl in pods can be controlled via annotations on the PodSecurityPolicy.
+The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
 
-Here is an example, it authorizes binding user creating pod with corresponding
-_safe_ and _unsafe_ sysctls.
+Sysctl annotations represents the key of a whitelist of allowed safe and unsafe sysctls
```
Comment (php-coder) [inline docs/concepts/cluster-administration/sysctl-cluster.md]: s/annotations/annotation/

s/represents the key of a whitelist of/represents a whitelist of/ ?

**Comment by WanLinghao** [comment]
@php-coder PTAL

**Comment by bradtopol** [comment]
@zacharysarah Makes sense! Thanks for the clarification.  Great job @WanLinghao 
/lgtm
@zacharysarah this ones ready for you

**Comment by php-coder** [comment]
@WanLinghao LGTM, thank you!

**Pull request merged**